### ISSUE TITLE
Discover Feed Doesn't Appear to Be Collecting All Emojis

### ISSUE THREAD

**Issue by jmillerv** [microdotblog/issues#230]
I'm trying to use the discover feed to, well,, discover new people and when I try to use say writing or books it only has posts from several hours ago. 

Does this collection of posts happen on a schedule or is it just not doing it. For example at the time of this writing (4:52 PM Mountain) when I refresh the writing feed, the latest post is from 3:49 AM and if I do the same in books, the post is from 7:12 AM. 

Is this intended behavior?

**Comment by manton** [comment]
Not intended, that sounds like a bug or delay. For the emoji topics like books or writing, Micro.blog should be collecting them automatically when it processes people's blog posts. Do you happen to have a blog post URL that was missed?

**Comment by jmillerv** [comment]
So, when I go to the writing collection the last post is from 9-29-21 at 4:52 PM.

But I posted a really short post with the writing emoji last night at 10:28 PM 
 https://againstthefuture.net/2021/09/23/203650.html

**Comment by jmillerv** [comment]
I just noticed though that if I go to jmiller.micro.blog my page hasn't been updated since I added a domain name. Is there a misconfiguration here or would this be a separate issue? 

I wonder if this also coincides with another issue where I haven't been getting new badges. Like the 10 posts badge. 

But it looks like much of the micro.blog functionality stopped working when I added a domain name.

**Comment by jmillerv** [comment]
Not sure if this helps but I keep seeing an error like this in my logs for every post. 

```
2021-10-01 18:52:01: Feed: 304 not modified for 106875, https://jmiller.micro.blog/feed.json
2021-10-01 18:51:59: Feed: Downloading feed 106875, URL https://jmiller.micro.blog/feed.json
```

**Comment by jmillerv** [comment]
Alright, so, I'm not sure if anything's change on the micro code but my issue of not getting new content seems to be resolved in the discover section. 

As for my things not getting pulled in based on emojis, I think this was fixed by adding the feed from my custom domain name into the feeds section of my account. Also, as soon as I did that I noticed my github archive functionality started working too. 

Not sure what the solution is but I'm going to go ahead and close this issue since things seem to be working.
